qiskit/pulse/utils.py

```diff
@@ -38,25 +40,31 @@ def align_measures(schedules: List[ScheduleComponent], cmd_def: CmdDef) -> Sched
     Args:
         schedules: Collection of schedules to be aligned together
         cmd_def: Command definition list
+        cal_gate: The name of the gate to inspect for the calibration time
+        max_calibration_duration: If provided, cmd_def and cal_gate will be ignored
+        align_time: If provided, this will be used as final align time.
     Returns:
         Schedule
     Raises:
         ValueError: if an acquire or pulse is encountered on a channel that has already been part
                     of an acquire
     """
-    # Need time to allow for calibration pulses to be played for result classification
-    max_calibration_duration = 0
-    for qubits in cmd_def.cmd_qubits('u3'):
-        cmd = cmd_def.get('u3', qubits, np.pi, 0, np.pi)
-        max_calibration_duration = max(cmd.duration, max_calibration_duration)
-
-    # Schedule the acquires to be either at the end of the needed calibration time, or when the
-    # last acquire is scheduled, whichever comes later
-    acquire_scheduled_time = max_calibration_duration
-    for schedule in schedules:
-        last_acquire = max([time for time, inst in schedule.instructions
-                            if isinstance(inst, AcquireInstruction)])
-        acquire_scheduled_time = max(acquire_scheduled_time, last_acquire)
+    assert align_time is None or align_time >= 0, "Align time cannot be negative."
```

Comment:
The qiskit error handling policy is to raise a `QiskitError` or `PulseError` rather than use an `assert`.